compile.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+echo -e "\n\n\tMicrosoft RP2040 LampArray Sample run\n\n"
+
+
+echo -e "\n\tStep 2.\n"
+sudo git clone https://github.com/microsoft/RP2040MacropadHidSample.git
+
+echo -e "\n\tStep 3.\n"
+sudo apt install build-essential
+
+echo -e "\n\tStep 4\n"
+sudo apt install cmake gcc-arm-none-eabi libnewlib-arm-none-eabi libstdc++-arm-none-eabi-newlib
+sudo git clone https://github.com/raspberrypi/pico-sdk.git
```

Comment:
sudo shouldn't be needed for these

Reply:
Lines 14-16

Reply:
You need sudo for these 
![image](https://github.com/microsoft/RP2040MacropadHidSample/assets/92652074/7ccb415a-fe0c-44ff-b8a9-05b5182e0453)

Reply:
maybe even for lines 33 and 34

Reply:
Removed all sudos in the script, but when running the script it needs to be ran with `sudo ./setup.sh`